extensions/wikia/WikiFactory/tests/WikiFounderIpControllerTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * @group Integration
+ */
+class WikiFounderIpControllerTest extends WikiaDatabaseTest {
+	const WIKI_ID = 2;
+
+	/** @var RequestContext $requestContext */
+	private $requestContext;
+
+	/** @var WikiFounderIpController $wikiFounderIpController */
+	private $wikiFounderIpController;
+
+	protected function setUp() {
+		parent::setUp();
+		$dbw = WikiFactory::db( DB_SLAVE );
+
+		$dbw->insert( "city_list", [
+				"city_id" => self::WIKI_ID,
+				"city_url" => "http://test.wikia.com",
+				"city_sitename" => "wikdia",
+				"city_vertical" => 5,
+				"city_cluster" => "c1",
+				"city_dbname" => "unittests",
+				"city_founding_ip_bin" => inet_pton( "149.6.28.163" ),
+			], __METHOD__ );
```

Comment:
Please move this row to a YAML fixture, e.g. - https://github.com/Wikia/app/blob/2a4a827f0ace3c942493e0db42d669656961aef7/extensions/wikia/WikiFactory/tests/fixtures/api_variable.yaml#L2